examples/security/idcs-login/src/main/resources/application.yaml

```diff
@@ -46,11 +46,13 @@ security:
         client-id: "${ALIAS=security.properties.idcs-client-id}"
         client-secret: "${ALIAS=security.properties.idcs-client-secret}"
         identity-uri: "${ALIAS=security.properties.idcs-uri}"
-      cache-config:
-        cache-timeout-millis: 100000
   web-server:
     # protected paths on the web server - do not include paths served by Jersey, as those are protected directly
     paths:
-    - path: "/rest/profile"
-      methods: ["get"]
-      authenticate: true
+      - path: "/rest/profile"
```

Comment:
no idea whether this is defined by MicroProfile or native Helidon, but why don't you accept an array of paths here to use the same configuration multiple times?

Reply:
That is not implemented (and it is Helidon SE security integration).
If you feel this is useful, you can submit an enhancement issue.

Reply:
will do